### DISCUSSION TITLE
Should timelapse work with both picture and movie output?

### DISCUSSION THREAD

**Discussion by strips** [Motion-Project/motionplus#124] [Q&A]
Hi, every time I enable timelapse Motionplus start failing and restarting for every snapshot. Thought maybe I should ask about the functionality this time before reporting a possible bug 😄 

I'm running MotionPlus 0.1.1-git-20240108-ff5d572

This is all thats printet in the log even with DEBUG:
```
Jan 18 23:07:29 hakkebo motionplus[822554]: [NTC][ALL][00:ml00:Hakkebo1] mlp_init: Camera 1 started: motion detection Enabled
Jan 18 23:07:40 hakkebo systemd[1]: motionplus.service: Main process exited, code=killed, status=11/SEGV
Jan 18 23:07:40 hakkebo systemd[1]: motionplus.service: Failed with result 'signal'.
Jan 18 23:07:45 hakkebo systemd[1]: motionplus.service: Scheduled restart job, restart counter is at 7.
```

This is my timelapse config in `camera1.conf`
```
timelapse_interval 20
timelapse_mode daily
timelapse_fps 60
timelapse_container mpeg4
timelapse_filename %Y%m%d-timelapse
```

To simplify the issue I commented all the timelapse lines except this one
```
timelapse_interval 20
```

Regards
Stian

**Comment by miorapkmn** [comment]
I'm having the same issue here, though I didn't know it's related to the timelapse feature (but now that I think of it, it makes perfect sense).

**Comment by strips** [comment]
@Mr-Dave 
Do you have any input? Should I create a bug issue?

**Comment by Mr-Dave** [comment]
This is a bug.  I'll push a fix shortly.

Edit:  Resolved via 9065764
